txpipe/utils/metacal.py

```diff
@@ -13,7 +13,52 @@ def metacal_band_variants(bands, *names):
         for name in names
     ]
 
-def apply_response(R,g1,g2):
-    R11 = R[0][0]
-    R22  = R[1][1]
-    return g1/R11, g2/R22
\ No newline at end of file
+def calculate_selection_bias(g1, g2, sel_1p, sel_2p, sel_1m, sel_2m, delta_gamma):
+    import numpy as np
+    
+    S = np.ones((2,2))
+    S_11 = (g1[sel_1p].mean() - g1[sel_1m].mean()) / delta_gamma
+    S_12 = (g1[sel_2p].mean() - g1[sel_2m].mean()) / delta_gamma
+    S_21 = (g2[sel_1p].mean() - g2[sel_1m].mean()) / delta_gamma
+    S_22 = (g2[sel_2p].mean() - g2[sel_2m].mean()) / delta_gamma
+    
+    # Also save the selection biases as a matrix.
+    S[0,0] = S_11
+    S[0,1] = S_12
+    S[1,0] = S_21
+    S[1,1] = S_22
+    
+    return S
+
+def calculate_multiplicative_bias(g1_1p,g1_2p,g1_1m,g1_2m,g2_1p,g2_2p,g2_1m,g2_2m,sel_00,delta_gamma):
```

Comment:
They're both "multiplicative" biases, but I can't think of a better name for this.  "Response bias"?

Also I'm wondering if there's something we could do to simplify passing around these long lists of arguments.  Let's think about that later though.

Reply:
In general this stage has some large arguments being passed around and long lines of similar actions, I can look into cleaning it up for the the future.